sdk/python/tests/test_TransactionDescriptorProcessor.py

```diff
@@ -141,6 +141,42 @@ def test_can_lookup_value_with_zero_value(self):
 		# Assert:
 		self.assertEqual(0, deadline)
 
+	def test_can_lookup_value_when_value_is_list(self):
+		# Arrange:
+		key = 'flags'
+		value = [1, 2, 3]
+		processor = self._create_processor({key: value})
+
+		# Act:
+		result = processor.lookup_value(key)
+
+		# Assert:
+		self.assertEqual(result, [processor.type_converter(item) for item in value])
```

Comment:
can you remove call to type_converted, so that expected value is explicit 

also, would be good to swap the assertEqual arguments for consistency